README.md

```diff
@@ -16,6 +16,19 @@ On a mac, run this:
 
 `zsh webserver-darwin.sh`
 
-on linux runn this:
+On arch linux run this:
 
 `bash webserver-linux.sh`
+
+On debian or ubuntu linux run this:
```

Comment:
I like this, it will force us to make a decision. Especially given the idea this week. That the README should act as a outline.